source/includes/steps-create-agg-pipeline.yaml

```diff
@@ -64,5 +64,17 @@ content: |
 
          .. figure:: /images/compass/agg-builder-exclude-stage-example.png
             :alt: Aggregation Builder exclude stage example
+---
+title: Optional. Use custom collation.
+level: 4
+ref: use-custom-collation
+content: |
+   a. Click the :guilabel:`Expand` icon to the left of the
+      :guilabel:`Folder` icon.
+
+      .. figure:: /images/compass/agg-builder-collation-stage-example.png
+            :alt: Aggregation Builder collation stage example
+
+   b. Enter the desired :manual:`collation </reference/collation/>`.
```

Comment:
Suggest: Enter the desired collation *document*.

Reply:
Fixed.